### PR TITLE
[SMALLFIX]Supply the variable name to Preconditions.checkNotNull

### PR DESCRIPTION
**Pull request by Popeye13**

### CODE REVIEW

**Comment by gpang** [comment]
Jenkins, test this please

File: core/server/common/src/main/java/alluxio/master/AbstractMaster.java [hunk 1]
```diff
@@ -86,9 +86,9 @@
    */
   protected AbstractMaster(Journal journal, Clock clock,
       ExecutorServiceFactory executorServiceFactory) {
-    mJournal = Preconditions.checkNotNull(journal);
-    mClock = Preconditions.checkNotNull(clock);
-    mExecutorServiceFactory = Preconditions.checkNotNull(executorServiceFactory);
+    mJournal = Preconditions.checkNotNull(journal,"journal");
```
Comment (gpang) [inline core/server/common/src/main/java/alluxio/master/AbstractMaster.java]: Please add a space after the comma.

File: core/server/common/src/main/java/alluxio/master/AbstractMaster.java [hunk 2]
```diff
@@ -86,9 +86,9 @@
...
+    mClock = Preconditions.checkNotNull(clock,"clock");
```
Comment (gpang) [inline core/server/common/src/main/java/alluxio/master/AbstractMaster.java]: Please add a space after the comma.

File: core/server/common/src/main/java/alluxio/master/AbstractMaster.java [hunk 3]
```diff
@@ -86,9 +86,9 @@
...
+    mExecutorServiceFactory = Preconditions.checkNotNull(executorServiceFactory,"executorServiceFactory");
```
Comment (gpang) [inline core/server/common/src/main/java/alluxio/master/AbstractMaster.java]: Please add a space after the comma.

**Comment by apc999** [comment]
Please fix the check style 
@Popeye13 
/home/jenkins/workspace/Alluxio-Pull-Request-Builder/core/server/common/src/main/java/alluxio/master/AbstractMaster.java:91: error: Line is longer than 100 characters (found 107).

File: core/server/common/src/main/java/alluxio/master/AbstractMaster.java [hunk 4]
```diff
@@ -86,9 +86,9 @@
    */
   protected AbstractMaster(Journal journal, Clock clock,
       ExecutorServiceFactory executorServiceFactory) {
-    mJournal = Preconditions.checkNotNull(journal);
-    mClock = Preconditions.checkNotNull(clock);
-    mExecutorServiceFactory = Preconditions.checkNotNull(executorServiceFactory);
+    mJournal = Preconditions.checkNotNull(journal, "journal");
+    mClock = Preconditions.checkNotNull(clock, "clock");
+    mExecutorServiceFactory = Preconditions.checkNotNull(executorServiceFactory, "executorServiceFactory");
```
Comment (apc999) [inline core/server/common/src/main/java/alluxio/master/AbstractMaster.java]: too long

**Comment by yupeng9** [comment]
@Popeye13 thanks for the contribution. Please reopen this PR towards master branch.

**Pull request closed without merging**